Constitution.txt

```diff
@@ -1,490 +1,264 @@
-That after such Publication the Electors should be appointed, and the Senators and Representatives elected: That the Electors should meet on the Day fixed for the Election of the President, and should transmit their Votes certified, signed, sealed and directed, as the Constitution requires, to the Secretary of the United States in Congress assembled, that the Senators and Representatives should convene at the Time and Place assigned; that the Senators should appoint a President of the Senate, for the sole Purpose of receiving, opening and counting the Votes for President; and, that after he shall be chosen, the Congress, together with the President, should, without Delay, proceed to execute this Constitution.
+2: If the exiting State has land borders with a territory which is not a member of the Union, or has water access to the Oceans of the World through waters not claimed as territory by any other State of the Union, the State shall be considered to be sovereign and foreign from these United States on the date one year after the President of these United States has been informed of the State's intent to leave the Union.
 
- 	
-By the unanimous Order of the Convention
+3: Otherwise, the exiting state shall be considered sovereign and foreign on the date two years after the President of these United States has been informed.
 
-W. Jackson  Secretary.
+4: During the period of time between the election of the State to leave the Union and the date of exit, the government of the State shall engage in negotiation of treaties to be activated on the date of exit, to govern trade between the newly separated State and the Union, and to come to agreement regarding the ownership of such lands within the borders of the newly separated State which previously had been purchased for use by the Government of these United States. For the purpose and within the scope of these negotiations, the State shall immediately be considered sovereign and foreign from these United States.
 
-Go: Washington -Presidt.
+5: A State which so elects to leave the Union shall be treated as a friend and neighbor unless the State's actions shall decry such treatment; but the State shall be forbidden from rejoining the Union for a period of twenty-four years.
```

Comment:
There should be no illusions about what an Exit means: the State will be separate from the Union. This cooling off period will allow the State to rejoin, if full sovereignty turns out to be a poor decision, but it makes it clear that the Union has no claim to force them to rejoin, and that the State cannot play an in-again, out-again game on a regular basis.